src/js/modules/comments/actions.js

```diff
@@ -70,7 +70,7 @@ export const createComment = values => {
 export const fetchComments = () => {
   return (dispatch) => {
     dispatch({ type: t.FETCH_COMMENTS_PENDING });
-    axios.get(`${STUY_SPEC_API_URL}/comments`, STUY_SPEC_API_HEADERS)
+    axios.get(`${STUY_SPEC_API_URL}/articles`, STUY_SPEC_API_HEADERS)
```

Comment:
Use the real URL, since you're only testing if your login modal can work. I say this because, on my side of the branch, I'm actively testing with my local rails server.